app/views/core/JuniorClassicChoiceModal.js

```
@@ -0,0 +1,65 @@
+let JuniorClassicChoiceModal
+require('app/styles/modal/junior-classic-choice-modal.sass')
+const ModalView = require('views/core/ModalView')
+const template = require('app/templates/core/junior-classic-choice-modal')
+const storage = require('core/storage')
+
+// define expectations for good rates before releasing
+
+module.exports = (JuniorClassicChoiceModal = (function () {
+  JuniorClassicChoiceModal = class JuniorClassicChoiceModal extends ModalView {
+    static initClass () {
+      this.prototype.id = 'junior-classic-choice-modal'
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)